Tasks/google-play-release/GooglePlay.ts

```diff
@@ -2,7 +2,7 @@ import fs = require('fs');
 import path = require('path');
```

Comment:
It appears that the PowerShell version of this task (GooglePlay.ps1) merely passes input values to the Node.js version.  Can we delete GooglePlay.ps1 and remove its reference in task.json?